pycbc/distributions/sky_location.py

```diff
@@ -243,6 +327,24 @@ def normalize_azimuth(self, phi):
         phi[phi >= 2 * numpy.pi] -= 2 * numpy.pi
         return phi
 
+    def to_uniform_patch(self, coverage):
+        """Apply the coverage criterion on the skymap
+        """
+        non_zero_ind = numpy.flatnonzero(self.pix_probs)
+        dtype = numpy.dtype([('index', numpy.ndarray), ('prob', numpy.float64)])
+        prob = self.pix_probs[non_zero_ind]
+        ind_prob = numpy.array(list(zip(non_zero_ind, prob)), dtype=dtype)
+        ind_prob_sorted = numpy.sort(ind_prob, order='prob')
+        prob = numpy.sort(prob)
+        ind_prob_rev_sorted = ind_prob_sorted[::-1]
+        list_ind = []
+        for i in range(len(ind_prob_sorted)):
+            list_ind.append(ind_prob_rev_sorted[i][0])
+        cum_sum_prob = numpy.cumsum(prob[::-1])
+        covered_sky = cum_sum_prob[cum_sum_prob<coverage]
+        self.pix_probs[list_ind[:len(covered_sky)]] = 1/len(covered_sky)
+        self.pix_probs[self.pix_probs!= 1/len(covered_sky)] = 0
+
```

Comment:
Not super necessary (maybe to be done in a later PR) but I would prefer this method to return a *new* HealpixSky instance, rather than modifying the existing instance, for consistency with the same method of the FisherSky class.